ios/Classes/Bindings/Models/ConferenceDTO.swift

```diff
@@ -323,4 +323,14 @@ extension DTO {
     struct AudioProcessingSenderOptions: Codable {
         let audioProcessing: Bool?
     }
+    
+    struct MessageReceivedData: Codable {
+        let message: String
+        let participant: Participant
+        
+        init(message: String, participant: VTParticipant) {
+            self.message = message
+            self.participant = DTO.Participant(participant: participant)
+        }
+    }
```

Comment:
This class is used in the command service. Do we have a file for the Command related DTOs? If not could we create one?

Reply:
We don’t have it. I will create one.